tracer/src/Datadog.Trace/Iast/Mark.cs

```diff
@@ -0,0 +1,17 @@
+// <copyright file="Mark.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+
+#nullable enable
+
+namespace Datadog.Trace.Iast;
+
+[Flags]
+internal enum Mark : byte
```

Comment:
I'd have chosen a more explicit name, like `SecureMarks` (plural because it's a flag enum)

Reply:
Changed in 6f642d9